suite-common/wallet-utils/src/solanaUtils.ts

```diff
@@ -26,6 +27,20 @@ const loadSolanaSystemProgramLib = async () => {
 const loadSolanaTokenProgramLib = async () => {
     return await import('@solana-program/token');
 };
+const loadSolanaToken2022ProgramLib = async () => {
+    return await import('@solana-program/token-2022');
+};
+
+const loadConcreteTokenProgramLib = (tokenProgramName: TokenProgramName) => {
```

Comment:
np: i guess we could inline the `loadSolanaTokenProgramLib` and `loadSolanaToken2022ProgramLib` functions and rename this to `loadSolanaTokenProgramLib`.